rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewKeyEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.jakewharton.rxbinding.view;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.KeyEvent;
+import android.view.View;
+
+/**
+ * A key event on a view.
+ * <p>
+ * <strong>Warning:</strong> Instances keep a strong reference to the view. Operators that cache
+ * instances have the potential to leak the associated {@link Context}.
+ */
+public final class ViewKeyEvent extends ViewEvent<View> {
+  private final int keyCode;
+  private final KeyEvent keyEvent;
```

Comment:
I guess you're right. Upon consulting `View`'s source  it seems like `keyCode` is there just for convenience reasons:

    ListenerInfo li = mListenerInfo;
    if (li != null && li.mOnKeyListener != null && (mViewFlags & ENABLED_MASK) == ENABLED
            && li.mOnKeyListener.onKey(this, event.getKeyCode(), event)) {
        return true;
    }

Given this, I think it's better for us to directly return `KeyEvent` instead.